src/db/models/BreachNotificationSubscriber.ts

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { SubscriberRow } from "knex/types/tables";
+import { getEmailAddressesByHashes } from "../tables/emailAddresses";
+import { getSubscribersByHashes } from "../tables/subscribers";
+
+// Covered by integration tests but report is not combined
+// TODO: MNTOR-5117
+/* c8 ignore start */
+export type BreachNotificationSubscriber = {
+  subscriber_id: SubscriberRow["id"];
+  all_emails_to_primary: SubscriberRow["all_emails_to_primary"];
+  primary_email: SubscriberRow["primary_email"];
+  breached_email: string;
+  notification_email: string;
```

Comment:
Nit, if we want to be really explicit:


```suggestion
  breached_email: SubscriberRow["primary_email"] | EmailAddressRow["email"];
  notification_email: SubscriberRow["primary_email"] | EmailAddressRow["email"];
```